ansible/roles/jupyterhub/files/jupyterhub_config.py

```diff
@@ -106,10 +106,10 @@
 )
 
 # User authentication class
-c.JupyterHub.authenticator_class = 'firstuseauthenticator.FirstUseAuthenticator'
+# c.JupyterHub.authenticator_class = 'firstuseauthenticator.FirstUseAuthenticator'
```

Comment:
We should leave the `firstuseauthenticator` as the default authenticator.

Reply:
I agree.